npm/webpack-preprocessor/package.json

```
@@ -47,7 +47,7 @@
   "peerDependencies": {
     "@babel/core": "^7.25.2",
     "@babel/preset-env": "^7.25.3",
-    "babel-loader": "^8.3 || ^9",
+    "babel-loader": "^8.3 || ^10",
```

This looks like it would be a breaking change if `^9` is removed

---

The breaking changes for from babel-loader [v10](https://github.com/babel/babel-loader/releases) are:
* bump node requirement to ^18.20.0 || ^20.10.0 || >=22.0.0 and webpack requirement to >= 5.61.0
* breaking: use output.hashFunction as loader cache hasher

`@cypress/wepback-preprocessor` uses webpack `5.88.2` and the peer dependencies are set to `"webpack": "^4 || ^5"`.

The [PR](https://github.com/babel/babel-loader/pull/1027) for the 2nd babel-loader change includes this note:
```
* Impact: Low
* Migration path for existing applications: purge the babel-loader cache before upgrading babel-loader. If they are not purged, the function of babel-loader will not be impacted but doing so will save some disk space.
```

`@cypress/wepback-preprocessor` seems to be compatible with these changes. Is there something else I should check/test?

---

@cek-roadmunk-tempo @MikeMcC399 is correct here in that this would be a breaking change to the package and to Cypress as it error for users still on `babel-loader` 9.
```suggestion
    "babel-loader": "^8.3 || ^9 || ^10",
```

---

@AtofStryker Thanks for clarifying. I commited your suggestion.